### ISSUE TITLE
Drag option should be available for all devices on Kanban

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1811]
Currently when I have multiple boards and drag across the screen, nothing happens(I would have to use the scrollbar to see other boards). However on a mobile device that is possible and it would be great if this behaviour is available on all devices
